src/test/java/com/funeat/product/persistence/ProductRepositoryTest.java

```diff
@@ -189,4 +193,100 @@ class findAllWithReviewCountByNameContaining_성공_테스트 {
                     .isEqualTo(expected);
         }
     }
+
+    @Nested
+    class findAllByTagFirst_성공_테스트 {
+
+        @Test
+        void 특정_태그가_포함된_상품들을_조회한다() {
+            // given
+            final var category = 카테고리_간편식사_생성();
+            단일_카테고리_저장(category);
+
+            final var 태그_맛있어요 = 태그_맛있어요_TASTE_생성();
+            final var 태그_단짠단짠 = 태그_맛있어요_TASTE_생성();
+            final var 태그_갓성비 = 태그_맛있어요_TASTE_생성();
```

Comment:
이 부분이랑 `특정_태그와_마지막_상품아이디_이후_상품들을_조회한다` 테스트 모두 이름에 맞는 태그로 변경하는게 더 정확해보여요

```suggestion
            final var 태그_맛있어요 = 태그_맛있어요_TASTE_생성();
            final var 태그_단짠단짠 = 태그_단짠단짠_TASTE_생성();
            final var 태그_갓성비 = 태그_갓성비_PRICE_생성();
```

Reply:
이거 아직 수정 안됐어요~